src/mainPrism.js

```diff
@@ -150,8 +151,10 @@ class MainPrism extends React.Component {
   render() {
     const { sceneIndex } = this.state;
     const scene = this.scenes[sceneIndex];
+    const size = this.props.size ? this.props.size : defaultProps.size;
+    const position = this.props.position ? this.props.position : defaultProps.position;
     return (
-      <Prism size={this.props.size}>
+      <Prism size={size} position={position}>
```

Comment:
We don't need to check if this.props.{value} exists, **defaultProps** do it for us.